desktop-widgets/tab-widgets/TabDiveInformation.cpp

```diff
@@ -220,7 +223,7 @@ void TabDiveInformation::updateData()
 	ui->atmPressType->setCurrentIndex(0);                // Set the atmospheric pressure combo box to mbar
 	salinity_value = get_dive_salinity(current_dive);
 	if (salinity_value) {			// Set water type indicator (EN13319 = 1.020 g/l)
-		if (prefs.salinityEditDefault) {   //If edit-salinity is enabled then set correct water type in combobox:
+		if (ui->waterTypeCombo->isVisible()) {   // If water salinity is editable then set correct water type in combobox:
```

Comment:
Looks like this is also fixing a bug here, where the salinity isn't set correctly for manual dives. Good find!